ocs_ci/helpers/performance_lib.py

```diff
@@ -209,3 +210,82 @@ def measure_pvc_creation_time(interface, pvc_name, start_time):
         f"Creation time (in seconds) for pvc {pvc_name} is {(et - st).total_seconds()}"
     )
     return (et - st).total_seconds()
+
+
+def diff_check(first, second, diffs=10):
+    """
+    Function to check the differance between 2 numbers, and return true if the
+    difference between them is more then expected
+
+    Args:
+        first (int) : the first number (usual the lowest)
+        second (int) : the second number (usual the highest)
+        diffs (int) : the acceptable difference between the number in percentage
+           the default is 10%
+
+    Return:
+        bool : True if the difference is more then the acceptable, other False
+
+    """
+    try:
+        # using the abs since the first number can be higher then the second
+        # one, and in this case the results can be negative
+        if abs((100 - (first * 100 / second))) > diffs:
+            return True
+    except ZeroDivisionError:
+        pass
+    return False
+
+
+def cleanup_results_numbers_from_spikes(data, spike=10):
```

Comment:
@Avilir IMO, this method should not be applied for all cases. 
For eg., PVC creation of 1s, 0.3s, 0.2s, 0.4s, and 0.3s would mean we are ignoring the 1s test result. This should not be ignored but it should be looked at why that is the case.